src/api/types.ts

```diff
@@ -202,3 +210,24 @@ export interface ContextResponse {
   }
   errors?: any
 }
+
+export interface ApplyContextResponseData {
+  id: string
+  type: ['broker_context']
+  relationships: {
+    integrations_relationships: [id: string, type: ['broker_integration'], org_id: string, integration_type: string]
```

Comment:
@aarlaud, should it be `broker-integration`?

Reply:
sorted.